### DISCUSSION TITLE
Bone control?

### DISCUSSION THREAD

**Discussion by lawlerboy1** [vrchat-community/osc#112] [Q&A]
Hey so I've been trying to recreate the hand trackers that someone else made using OSC, and I've almost got my hand tracker done, just one problem, how do I control bones? I already have all the location values I just need to control the endpoints of bones.

**Comment by momo-the-monster** [comment]
We don't currently expose an endpoint for controlling bones, sorry.l

**Comment by vgaggia** [comment]
Hello, just wondering if there are any updates on this?
I have an external object on my avatar that would be perfect if it could be tracked with a tracker in real life

**Comment by jameswilddev** [comment]
Same here - could we maybe make a single-frame animation in all eight corners of the box in which the tracker might be, then use OSC animation parameters to blend between them?
